config/locales/generic_work.en.yml

```diff
@@ -38,6 +38,7 @@ en:
         subject: "Topical terms describing the “aboutness” of the work. Depicted persons, places, and time periods are covered here as well. Field supports FAST (derived from Library of Congress authorities) autocomplete. <strong><em>Ex: • Alchemy • Integrated circuits</strong></em>. Recommended."
         division: "Select the CHF administrative entity responsible for the management and curation of the physical work. Required."
         exhibition: "Select any current or past Museum exhibition(s) that the work has been displayed in, if applicable. If a desired entry is not available, contact the Digital Projects and Metadata Librarian."
+        project: "Project metadata instructions (TBD)"
```

Comment:
I think I saw these go by somewhere from @yonyitz , do we have em to include before we merge?

Reply:
They're in there.... could you be looking at an out-of-date version of the pr?

Reply:
I was!